FunctionalEquivalence/subworkflows/F1Evaluation.wdl

```diff
@@ -107,126 +85,131 @@ def read_file(data, filename, stratifiers):
     else:
         var_type = 'snp'
     
-    f1_data = read_roc_to_f1_dict(filename)
-    
-    if dataset not in data:
-        data[dataset] = dict()
-    if replicate not in data[dataset]:
-        stratifier_dict = {'tool1': dict(), 'tool2': dict()}
-        for stratifier in stratifiers:
-            stratifier_dict['tool1'][stratifier] = dict()
-            stratifier_dict['tool2'][stratifier] = dict()
-        
-        data[dataset][replicate] = stratifier_dict
-    data[dataset][replicate][tool][region][var_type] = f1_data
-
-def extract_data_from_dict(score_range, data, dataset, tool, region, var_type):
-    return [
-                np.array([
-                    data[dataset][replicate][tool][region][var_type][score] if score in data[dataset][replicate][tool][region][var_type] else np.nan
-                    for replicate in data[dataset].keys()
-                ])
-                for score in score_range
-            ]
-    
+    file_data = pd.read_csv(filename, header=6, delimiter='\t')
```

Comment:
The magic 6 here looks like Picard files to me. Regardless, I think if you're trying to skip some headers/comments at the top, try using the `comment` input field, e.g. `comment='#'` to automatically skip any lines beginning with a `#` in favor of the hard-coded number of lines to skip.

Reply:
Unfortunately this doesn't work because annoyingly, the header line also starts with a `#` in these RTG vcfeval files. They look like this:
```
#comment
...
#comment
#score	true_positives_baseline ...
value2	value2
```